src/plugins/console/server/lib/spec_definitions/json/generated/cluster.pending_tasks.json

```diff
@@ -10,6 +10,6 @@
     "patterns": [
       "_cluster/pending_tasks"
     ],
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/cluster-pending.html"
+    "documentation": "https://opensearch.org/docs/latest/"
```

Comment:
For any of these that don't have OpenSearch documentation, can we add them to the `noDocumentation` object in the doc links service?

Reply:
Yes. wil add